front/components/sparkle/navigation.tsx

```diff
@@ -316,6 +317,13 @@ export const subNavigationAdmin = ({
           href: `/w/${owner.sId}/u/extract`,
           current: current === "extract",
         },
+        {
+          id: "tables",
```

Comment:
As per IRL this is temporary as there is no reason for a Tables page at all anymore. Tables should be along documents in folders.

Reply:
Yes ✅